sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -1980,6 +1993,7 @@ def _get_page_ranges_options( # type: ignore
         options.update(kwargs)
         return options
 
+    @check_parameter_api_version(managed_disk_diff='2019-04-19')
```

Comment:
>check_parameter_api_version [](start = 5, length = 27)

It seems we still want to keep the version check as a policy instead of a parameter so that we can still catch the improper operation when users override the version in the pipeline?
@amishra-dev Do you have any idea about the decision on removing the version check policy across all SDKs?

Reply:
If a customer is overriding the API version in the pipeline - then they should be able to manage their own requests without us enforcing client-side validation.

Reply:
Have confirmed that the policy approach will not be used across language SDKs.

Reply:
Can we hold off this pr for now? I just talked to java person and she hasn't got informed about this. Ashish is sick today, can we discuss this again when he's back ^_^